spanner/spanner_samples.rb

```diff
@@ -924,6 +971,29 @@ def query_with_parameter project_id:, instance_id:, database_id:
   # [END spanner_query_with_parameter]
 end
 
+def query_with_numeric_parameter project_id:, instance_id:, database_id:
+  # [START spanner_query_with_numeric_parameter]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+  require "bigdecimal"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+
+  sql_query = "SELECT VenueId, Revenue FROM Venues WHERE Revenue < @revenue"
+
+  params      = { revenue: BigDecimal("100000") }
+  param_types = { lastName: :NUMERIC }
```

Comment:
I think there is a typo here, shouldn't it be:

```ruby
params      = { revenue: BigDecimal("100000") }
param_types = { revenue: :NUMERIC } # this should be revenue, not lastName
```

Reply:
Nice catch.